what_the_fec/main.py

```diff
@@ -1,39 +1,61 @@
 import os
-from datetime import date
-from typing import Annotated
 
-from fastapi import Depends, FastAPI, Form, Request
-from fastapi.responses import HTMLResponse
+from fastapi import Depends, FastAPI
 from fastapi.staticfiles import StaticFiles
 from fastapi.templating import Jinja2Templates
-from sqlalchemy import Connection
-from sqlalchemy.sql import text
 
-from what_the_fec.api.candidate_office_records import (
-    delete_single_candidate_office_records_func,
-    delete_single_candidate_office_records_page_func,
-    edit_single_candidate_office_records_page_func,
-    get_all_candidate_office_records_func,
-    post_single_candidate_office_records_func,
-    update_single_candidate_office_records_func,
+from what_the_fec.dependencies import get_db_conn, get_templates, templates_init
+from what_the_fec.routes.amendment_indicators.routes import (
+    router as amendment_indicators_router,
 )
-from what_the_fec.api.election_years import (
-    get_all_election_years_func,
+from what_the_fec.routes.candidate_office_records.routes import (
+    router as candidate_office_records_router,
+)
+from what_the_fec.routes.candidate_office_records_committees.routes import (
+    router as candidate_office_records_committees_router,
+)
+from what_the_fec.routes.candidate_office_records_contributions.routes import (
+    router as candidate_office_records_contributions_router,
+)
+from what_the_fec.routes.candidates.routes import router as candidates_router
+from what_the_fec.routes.committees.routes import router as committee_types_router
```

Comment:
```suggestion
from what_the_fec.routes.committee_types.routes import router as committee_types_router
```

Suggestion will fix `"GET /committee_types HTTP/1.1" 404 Not Found` error when trying to access http://127.0.0.1:8000/committee_types/